tests/fixtures/common/collections/ansible_collections/testorg/coll_1/plugins/lookup/lookup_1.py

```diff
@@ -1,6 +1,8 @@
 """An ansible test lookup plugin."""
 
-from __future__ import absolute_import, division, print_function
```

Comment:
Do we run `ansible-test sanity` against these anywhere? If we do (or will), this'll break the boilerplate linting.

Reply:
I'm actually not entirely sure if it enforces all of these on the same line but it's worth checking.

Reply:
yeah, it complains......  skipping the file.

Reply:
So this should be reverted before it's resolved, then.

Reply:
ha, I reverted the module but missed the lookup! thanks

Reply:
I've reverted skipped the entire collections folder in the configuration, since it is just a fixture and all files would need to be excluded.  thanks for catching this. @webknjaz Can you resolve this?

Reply:
resolving conversation, please reopen if the concerns and question are unanswered, ty for the feedback

Reply:
Looks good. Thanks for letting me check this :)